functionnal_tests/cypress/integration/app_interface/switching_apps_feature_spec.js

```diff
@@ -0,0 +1,124 @@
+import { create_htmldocument } from '../helpers/htmldoc.js'
+import { create_file } from '../helpers/file.js'
+import { create_thread } from '../helpers/thread.js'
+
+describe('App Interface (the mechanism to open and close apps)', () => {
+  const htmlDocTitle = 'HtmlDocForSwitch'
+  const threadTitle = 'ThreadForSwitch'
+  const fileTitle = 'FileForSwitch'
+  let workspaceId
+
+  before(() => {
+    cy.resetDB()
+    cy.setupBaseDB()
+    cy.loginAs('administrators')
+    cy.fixture('baseWorkspace').as('workspace').then(workspace => {
+      workspaceId = workspace.workspace_id
+      cy.visit(`/ui/workspaces/${workspaceId}/contents`)
+
+      create_file(cy, fileTitle)
+      cy.get('[data-cy="popinFixed__header__button__close"]').should('be.visible').click()
+
+      create_htmldocument(cy, htmlDocTitle)
+      cy.get('[data-cy="popinFixed__header__button__close"]').should('be.visible').click()
+
+      create_thread(cy, threadTitle)
+      cy.get('[data-cy="popinFixed__header__button__close"]').should('be.visible').click()
+    })
+  })
+
+  describe('Switching between 2 different apps feature', () => {
+    const contentHtmlDocGetter = `.workspace__content__fileandfolder > .content[title="${htmlDocTitle}"]`
+    const contentThreadGetter = `.workspace__content__fileandfolder > .content[title="${threadTitle}"]`
+    const contentFileGetter = `.workspace__content__fileandfolder > .content[title="${fileTitle}"]`
+
+    beforeEach(() => {
+      cy.wait(200)
```

Comment:
add a `fixme` as we shouldn't need a wait

Reply:
fix me added